File: server/utils/get_summary.js
// 私以外私じゃないの
const https = require('https')

const data = JSON.stringify({
  todo: 'Buy the milk'
})

const options = {
  hostname: "api-3moji.herokuapp.com",
  port: 443, method: 'POST',
  path: "/api/v1/summary/",
  headers: {
    'Content-Type': 'application/json',
    'Content-Length': data.length
  },
}

const req = https.request(options, res => {
  if (res.statusCode !== 200) return console.log(`statusCode: ${res.statusCode}`);

  res.on('data', d => {
    process.stdout.write(d)
  })
})

req.on('error', error => {
  console.error(error)
})

req.write(data)
req.end()

File: utils/get_summary.go
package main

import (
  "io/ioutil"
  "net/http"
  "fmt"
)

func main() {
  resp, err := http.Post("https://api-3moji.herokuapp.com/api/v1/summary/", "", nil)
  if err != nil {
    fmt.Println(err)
    return
  }
  body, err := ioutil.ReadAll(resp.Body)
  if err != nil {
    fmt.Println(err)
    return
  }
  fmt.Printf("%s", body)
}
